molecule/manual-activemq/hosts.yml

```
@@ -0,0 +1,10 @@
+all:
+  hosts:
+  children:
+    activemq:
+      hosts:
+        whatever.mq.eu-west-1.amazonaws.com:
+          managed_host: true
```

```suggestion
          managed_host:
```

don't know if `external_host` is more explicit (managed by whom? ansible, the user or someone else?)